Gruntfile.js

```diff
@@ -1637,6 +1652,32 @@ module.exports = function(grunt) {
 		} );
 	} );
 
+	grunt.registerTask( 'packages:update', 'Update WordPress packages', function() {
```

Comment:
Maybe we should prefix the name with `wp-` to make it explicit we are dealing with WordPress packages:
```suggestion
	grunt.registerTask( 'wp-packages:update', 'Update WordPress packages', function() {
```

Also applicable to `:refresh-deps`.